charts/templates/apiserver-apiservice.yaml

```diff
@@ -0,0 +1,16 @@
+{{- $name := include "clusterpedia.name" . -}}
```

Comment:
I think that declaring template variables again in a separate file does not do any good and adds a degree of complexity.

Reply:
For a separate file that only needs to use the value once, this really doesn't make sense, but for a file that uses the value multiple times, it's much more readable.